Intersect.Client/Interface/Game/Typewriting/Typewriter.cs

```diff
@@ -0,0 +1,145 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Intersect.Client.Core;
+using Intersect.Client.Framework.Gwen.Control;
+using Intersect.Configuration;
+using Intersect.Utilities;
+
+namespace Intersect.Client.Interface.Game.Typewriting
+{
+
+    sealed internal class Typewriter
```

Comment:
The access modifiers for the class are reversed (?) I think that it should be `internal sealed` instead of `sealed internal`.

Access modifiers in C# follow a specific order. 1st comes the visibility modifier (public, protected, internal, or private), followed by the optional static, abstract, or sealed modifiers, and finally, any other modifiers like readonly, volatile, etc.

Reply:
Mkay cool, thanks for the info!